README.md

```diff
@@ -66,17 +66,17 @@ Best practice is, that we discuss (new) features and if they are needed. So don'
 ## ✨ Special thanks
 - ***SourRaindrop***: for creating a lot of custom images and assets like our logo
 
-## 🪛 Features that are missing to have the full osu!player plus feature set
+## 🪛 Features that are missing to have the full osu!player plus a feature set
 
 #### 🔧 Features with lower priority
 - [x] Audio-Equalizer 
 - [ ] Localization 
 - [ ] Language Editor 
 - [ ] Language Manager to upload and download translations   
 - [ ] Hotkey support  
-- [ ] Export songs to directory  
+- [ ] Export songs to the directory  
 - [ ] Custom themes 
-- [ ] Miniplayer to save some space  
+- [ ] Mini player to save some space  
```

Comment:
One word, because that's how it's called and know by the community. As it is a *name*

Reply:
resolved